db/seeds.rb

```diff
@@ -1,7 +1,38 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+Pet.destroy_all
```

Comment:
It's  hard to say what the error was that you got from Render when dealing with seeds, but my guess is that you deleted pets that were associated with an ApplicationPet and Application record(s). To avoid this error, you could first include ApplicationPet.destroy_all, and then proceed with these other deletes. That's because your ApplicationPet table has foreign keys to other tables so those need to be deleted first before the Pet or Application record is deleted. 

Another option is to check out `dependent: :destroy`